src/torcontrol.cpp

```diff
@@ -524,11 +524,15 @@ void TorController::auth_cb(TorControlConnection& _conn, const TorControlReply&
 
         // Now that we know Tor is running setup the proxy for onion addresses
```

Comment:
nit: while here, can you please touch this up to add a missing comma and fix "setup", which is a noun but not a verb:

`s/running setup/running, set up/`